src/main/host/process.c

```diff
@@ -500,6 +516,13 @@ static void _process_start(Process* proc) {
     descriptor_ref((LegacyDescriptor*)proc->stderrFile);
     g_free(stderrFileName);
 
+    if (proc->straceLoggingEnabled) {
+        gchar* straceFileName = _process_outputFileName(proc, "strace");
+        proc->straceFd = open(
+            straceFileName, O_CREAT | O_TRUNC | O_WRONLY, S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
```

Comment:
I *think* O_CLOEXEC would also be preferable here to avoid leaking the file descriptors into the managed processes. I see we're not using it for other files like stdout and stderr though and am trying to think whether there's a reason why.

I think even if we did want the shim to access these files directly, ideally we'd just clear the O_CLOEXEC flag for *that* process's files in between fork and exec, so that every process doesn't get every file descriptor from the managed processes that started before it.

Maybe it'd be better  to change all the files to use O_CLOEXEC in a separate PR though.

Reply:
Changed this file to be CLOEXEC.